pom.xml

```diff
@@ -689,6 +689,11 @@
         <version>3.0.1</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
```

Comment:
wrong location. we should put it earlier int he session where this is the "Provided scope" section